app/classes/document_query.rb

```diff
@@ -118,73 +118,80 @@ def suggestion_highlight
   def build_search_query
     #DSL reference: https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-dsl
     doc_query = self
+
     search.query do
-      bool do
-        if doc_query.query.present?
-          must do
-            bool do
-              #prefer bigram matches
-              should { match bigrams: { operator: 'and', query: doc_query.query } }
-              should { term  promote: true }
-
-              #prefer_word_form_matches
+      function_score do
```

Comment:
Note: The indentation change makes is look like way more has changed in this file. The  only actual changes  are the addition of:
```
  function_score do
    functions [{ gauss: { created: { origin: 'now', scale: '1825d', offset: '30d' } } }]
      #...
   end
```
...wrapping around the query.